README.md

```diff
@@ -35,9 +35,9 @@ them, are:
   at the top of the new issue / PR page instead of that link. Maintainers can
   choose to inline content there and / or link to other pages as appropriate.
 
-Hopefully none of these are a surprise to you as we’ve told you them before.
+Hopefully, none of these are a surprise to you, as we’ve told you them before.
 We’ve waited years now for progress on any of them. If GitHub were open source
-itself, we would be implementing these things ourselves as a community—we’re
+itself, we would be implementing these things ourselves as a community — we’re
```

Comment:
Hyphens?  Try a semicolon.  They were not invented for C syntax.  Hyphens are emotively used, but even non-liberal arts majors can learn to use semicolons.  😄